src/encoded/static/scss/encoded/modules/_static-pages.scss

```diff
@@ -640,7 +635,7 @@ $carousel-height : 240px;
 		font-size: 1.65rem;
 		cursor: pointer;
 		/* Works around glitch where parent slider has wrong height temp assigned and top: 50% isn't acccurate */
-		top: $carousel-height / 2 - 12px !important;
+		// top: $carousel-height / 2 - 12px !important;
```

Comment:
But is no longer quite vertically centered (is a bit lower than center) ->
![](https://i.gyazo.com/28807da986d6ebff1acef02b06823560.png) 

The refFunc should be already working around this by making the Carousel transparent/invisible until it becomes mounted...

Reply:
One 'CSS trick' to center vertically is `top: 50%` and then `margin-top: -10px` (or whatever 50% of height is)which might work here now that carousel-height is only relevant for Homepage..

Reply:
The default styling of Nuka Carousel component is top: 50%; so should be able to use that and just offset with a negative top margin

Reply:
Changed HomePage carousel to have icon-2x and changed to top: 50% and margin-top: -23px which seems to be working well. We have constraint now though that should always be `icon-2x` arrows. If anything we can add more conditional CSS styles (margin-top: -10px or something by default for reg size icons and if icon-2x is child, then margin-top the child another -13px or something).